modules/ROOT/pages/sync-gateway-channels.adoc

```diff
@@ -28,6 +28,7 @@ function (doc, oldDoc) {
   channel("foo");
 }
 ----
+The Sync Function must be surrounded with backticks (+`+) in the configuration file (xref::getting-started.adoc#configure-sync-gateway[one example]).
```

Comment:
>must

They can also be done with standard JSON double quotes, but in that case must have new lines escaped.